<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Ravi-PC" ignored="1" name="ChromeBrowser" tests="3" failures="1" timestamp="2020-04-26T21:54:49 IST" time="17.16" errors="0">
  <testcase name="register_user_with_valid_setof_data" time="2.938" classname="org.devlab.ui.testbed.RegistrationTest"/>
  <testcase name="Login_with_with_set_of_valid_data" time="3.278" classname="org.devlab.ui.testbed.LoginTest">
    <failure type="java.lang.AssertionError" message="Failed to login into application with valid set of user : validuser,validpwd expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed to login into application with valid set of user : validuser,validpwd expected [true] but found [false]
at org.devlab.ui.testbed.LoginTest.Login_with_with_set_of_valid_data(LoginTest.java:21)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- Login_with_with_set_of_valid_data -->
  <testcase name="Login_with_with_set_of_Invalid_data" time="3.293" classname="org.devlab.ui.testbed.LoginTest"/>
  <testcase name="@BeforeSuite beforeSuiteExc" time="0.0" classname="org.devlab.ui.basetest.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeSuite beforeSuiteExc -->
</testsuite> <!-- ChromeBrowser -->
